NetTopologySuite.IO/NetTopologySuite.IO.MsSqlSpatial/MsSqlSpatialReader.cs

```diff
@@ -9,7 +9,7 @@ public class MsSqlSpatialReader : WKBReader
         public override IGeometry Read(Stream stream)
         {
             var byteOrder = (ByteOrder)stream.ReadByte();

```

Comment:
Isn't this wrong now that the `Read(reader)` we do later will also try to read the byte order?

Reply:
Thanks! Fixed.